docs/source/notebooks/wps_bccaq_demo.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>Use <code>resource_filename</code>  for the <code>gcm_file</code> and   <code>obs_file</code>  to target the local paths.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/chickadee/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>